### PR TITLE
Hetzner refactor

### PR DESCRIPTION
**Pull request by kghamilton89**

### CODE REVIEW

File: Chart_of_the_Day/hetzner-outage/README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+If you're a [Hetzner](https://hetzner.de) hosting customer in Germany or elsewhere, you've likely found the service reliable and easy to manage. You're also likely to recommend it for its reasonable pricing with respect to bare computing and bandwidth services that Amazon and Google have yet to match. However, after the May 2018 power outage that affected all Hetzner primary datacenters outside Nuremberg, there remain lessons to be learned- especially for dedicated server customers.
 
 ## Infrastructure Outage
 
-An outage of this type typically affects shared resources at the datacenter level. Since DC07, DC10, DC12 source power from the same grid in Falkenstein, a problem with the grid affected all data centers in a similar fashion highlighting the grid as a single point of failure.
+An outage of this type typically affects shared resources at the datacenter level. Since servers `DC07`, `DC10`, and `DC12` source power from the same grid in Falkenstein, any problems that may occur will affect all data centers simultaneously, which highlights the grid as a single point of failure.
```
Comment (rodionos) [inline Chart_of_the_Day/hetzner-outage/README.md]: DCs are buildings stuffed with servers such as ours. So datacenters is correct here, not servers.

File: Chart_of_the_Day/hetzner-outage/README.md [hunk 2]
```diff
@@ -22,24 +22,24 @@ It was not starting any more so we had to replace it completely except for the d
 
 ## Hetzner Status
 
-The power outage was reported on May 24, 2018 at 09:30 UTC and affected a lot of customers, both physical as well as virtual.
+The power outage was reported on May 24, 2018 at 09:30 UTC and affected many customers, both physical and virtual.
```
Comment (rodionos) [inline Chart_of_the_Day/hetzner-outage/README.md]: It's probably better to say `, renting both dedicated and virtual servers.`

File: Chart_of_the_Day/hetzner-outage/README.md (hunk 1 above)
Comment (kghamilton89) [reply Chart_of_the_Day/hetzner-outage/README.md]: Changed.

File: Chart_of_the_Day/hetzner-outage/README.md (hunk 2 above)
Comment (kghamilton89) [reply Chart_of_the_Day/hetzner-outage/README.md]: Changed.

**Pull request merged**